contracts/KetherNFTRender.sol

```diff
@@ -21,26 +21,65 @@ contract KetherNFTRender is ITokenRenderer {
       '</g></svg>')));
   }
 
+  // Thanks to @townsendsam for giving us this reference https://gist.github.com/townsendsam/df2c420accb5ae786e856c97d13a2de6
+
+  function _generateAttributes(uint x, uint y, uint width, uint height, bool NSFW, bool forceNSFW) internal pure returns (string memory) {
+    return string(abi.encodePacked(
+      '[',
+         '{',
+            '"trait_type": "X",',
```

Comment:
Noting that we discussed calling this "Position X" in order to group them, but settled on "X".